app/views/agents_profile/index.html.haml

```diff
@@ -0,0 +1,3 @@
+.container.d-flex.flex-column.py-4{ style: "width: 1248px;" }
+  = render PageHeaderComponent.new(title: "Agents" , description: t('agents.description'))
+  = render TurboFrameComponent.new(id: 'agents-list', src: agents_path, loading: 'lazy')
```

Comment:
extract this into a helper (probably the agents_helpers) and reuse it in the admin/index page.